### PR TITLE
Add options for LVM cache specs to the 'logvol' command

### PR DESCRIPTION
**Pull request by vpodzime**
As the first step towards the LVM cache support we need to teach the parser to
accept the options for specifying cache size, PVs it should used and mode it
should use.

### CODE REVIEW

File: docs/kickstart-docs.rst [hunk 1]
```diff
@@ -1047,6 +1047,19 @@ Create a logical volume for Logical Volume Management (LVM).
     ``--size=``. This option must be used with
     ``--useexisting --size=``, or an error will be raised.
 
+``--cachesize``
+
+    Requested size of cache attached to the logical volume. (Requires --cachepvs)
```
Comment (bcl) [inline docs/kickstart-docs.rst]: What units does it expect?

File: pykickstart/handlers/f23.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ class F23Handler(BaseHandler):
         "lang": commands.lang.F19_Lang,
         "liveimg": commands.liveimg.F19_Liveimg,
         "logging": commands.logging.FC6_Logging,
-        "logvol": commands.logvol.F20_LogVol,
```
Comment (bcl) [inline pykickstart/handlers/f23.py]: Odd. So F21_LogVol wasn't used in F21?

File: docs/kickstart-docs.rst (hunk 1 above)
Comment (vpodzime) [inline docs/kickstart-docs.rst]: Good catch, thanks! It's megabytes as all our storage sizes, but I'm definitely going to make it explicit in the docs.

File: pykickstart/handlers/f23.py (hunk 2 above)
Comment (vpodzime) [inline pykickstart/handlers/f23.py]: *f21.py* has these set to ``F21_LogVol`` and ``F21_LogVolData``. But *f22.py* doesn't, so...oops! We should inspect these files and make sure the most recent versions are used in the future.

**Comment by clumens** [comment]
Merged by hand, thanks.

**Pull request closed without merging**